requirements/pip.txt

```diff
@@ -47,7 +47,7 @@ django-allauth==0.39.1
 GitPython==2.1.10  # pyup: ignore
 
 # Search
-elasticsearch==6.3.1
+elasticsearch==7.0.0
```

Comment:
This is a big one, we should test locally and see the changelog/migration path

Reply:
We first need to migrate to version 6.7 before updating to 7.0.
Source: https://www.elastic.co/blog/moving-from-types-to-typeless-apis-in-elasticsearch-7-0